### PR TITLE
[#2PlaysAMonth]: Calculator

### LINKED ISSUE #928: [#2PlaysAMonth]: Calculator
### What's the Play about?

I will create a Calculator which can perform basic arithmetic operations.

### What ReactJS concept will be used to create this play?

Components, Hooks

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by suryanshsoni120**
# Description

A Calculator which can perform basic arithmetic operations.

Fixes #928 

## Type of change

- [X] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Tested on local system.

# Checklist:

- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/plays/simple-calculator/SimpleCalculator.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import Calculator from './components/Calculator';
+
+// WARNING: Do not change the entry componenet name
+function SimpleCalculator(props) {
+  // Your Code Start below.
+  return (
+    <>
+      <div className="play-details">
+        <PlayHeader play={props} />
+        <div className="play-details-body">
```
Comment (atapas) [inline src/plays/simple-calculator/SimpleCalculator.js:12]: Add an additional and unique class name here, say `soni-calc`. We will use it in the CSS files.

File: src/plays/simple-calculator/styles/Buttons.css [hunk 2]
```diff
@@ -0,0 +1,54 @@
+.show-btn {
```
Comment (atapas) [inline src/plays/simple-calculator/styles/Buttons.css:1]: Please use the unique style as a prefix to all the class names here like `.soni-calc .equal`. THis is to reduce the chances of CSS overrides.

File: src/plays/simple-calculator/styles/Display.css [hunk 3]
```diff
@@ -0,0 +1,44 @@
+.display {
```
Comment (atapas) [inline src/plays/simple-calculator/styles/Display.css:1]: Same comment as another CSS

**Review by atapas — CHANGES_REQUESTED** [review]
Hey great work! Please take care of the review comments to get t merged.

File: src/plays/simple-calculator/Readme.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+# Simple Calculator
```
[the remark this answers is not in the document]
Comment (suryanshsoni120) [reply src/plays/simple-calculator/Readme.md:1]: Sir math.js library is used to perform the calculations.

File: src/plays/simple-calculator/SimpleCalculator.js (hunk 1 above)
Comment (atapas) [reply src/plays/simple-calculator/SimpleCalculator.js:12]: Hi @suryanshsoni120, please add the unique class name here and use it to prefix in your CSS file.

**Comment by atapas** [comment]
Hi @suryanshsoni120 Please catch me on discord as I still see the review comments opened about the style override. I can explain to you over a quick call.

**Comment by atapas** [comment]
@suryanshsoni120 Please add the video link

**Comment by atapas** [comment]
> @suryanshsoni120 Please add the video link

@suryanshsoni120 any update?

**Comment by atapas** [comment]
> > @suryanshsoni120 Please add the video link
> 
> @suryanshsoni120 any update?

@suryanshsoni120 will you please add the recording? Your good work will get merged.

**Comment by atapas** [comment]
@suryanshsoni120 I am merging your play. But it is still missing the video recording. Please update the video link before 28th Feb 2023 end of the day for it to be qualified for the event.

**Review by atapas — APPROVED** [review]

**Pull request merged**